hw/top_earlgrey/dv/verilator_sim_cfg.hjson

```diff
@@ -189,6 +189,14 @@
       name: usbdev_test
       sw_images: ["sw/device/tests/usbdev_test:1"]
     }
+    {
```

Comment:
Syntax looks good to me. You can try running:
```console
$ util/dvsim/dvsim.py hw/top_earlgrey/dv/verilator_sim_cfg.hjson -i sw_silicon_creator_lib_driver_hmac_functest sw_silicon_creator_lib_driver_uart_functest --max-parallel 1 --local
```

The `--max-parallel` by default is set to 16 (run 16 tests in parallel). This does not work for Verilator because the tests end up clobbering the UART on the host machine. The `--local` dispatches locally on your workstation as opposed to GCP.

Reply:
Thanks, the tests pass on my workstation.